docs/source/app-dev/ledger-api.rst

```diff
@@ -52,6 +57,11 @@ For full information on the services see :doc:`/app-dev/services`.
 
 You may also want to read the :doc:`protobuf documentation </app-dev/grpc/proto-docs>`, which explains how each service is defined as protobuf messages.
 
+As an alternative way to tap into the Ledger API you can also use the HTTP JSON API.
```

Comment:
"As an alternative way to tap into the Ledger API you can also use the HTTP JSON API." --> "You can also used the HTTP JSON API to tap into the Ledger API."

Apologies, the suggest feature doesn't seem to be available here.